dev_env/java_nginx_env/Dockerfile

```diff
@@ -1,9 +1,7 @@
 # This is the default base image for use with user models and workflows.
 FROM datarobot/java-dropin-env-base
 
-RUN apt-get update && apt-get install -y vim nginx
-
-RUN chmod 707 /var/lib/nginx
+RUN apt-get update && apt-get install -y vim
```

Comment:
why do we install vim?

Reply:
These are development/test/debug envs, so just to have it there.